lib/web/purchases_flutter_web.dart

```diff
@@ -0,0 +1,872 @@
+import 'dart:async';
+import 'dart:html' as html;
+import 'dart:js' as js;
+import 'package:flutter/services.dart';
+import 'package:flutter_web_plugins/flutter_web_plugins.dart';
+
+import '../errors.dart';
+
+const String rcPrefix = '\$rc_';
+
+class WebPackageType {
+  static const String unknown = 'unknown';
+  static const String custom = 'custom';
+  static const String lifetime = '${rcPrefix}lifetime';
+  static const String annual = '${rcPrefix}annual';
+  static const String sixMonth = '${rcPrefix}six_month';
+  static const String threeMonth = '${rcPrefix}three_month';
+  static const String twoMonth = '${rcPrefix}two_month';
+  static const String monthly = '${rcPrefix}monthly';
+  static const String weekly = '${rcPrefix}weekly';
+}
+
+enum LogLevel {
+  verbose,
+  debug,
+  info,
+  warn,
+  error,
+  silent,
+}
+
+class PurchasesFlutterPlugin {
+  static void registerWith(Registrar registrar) {
+    final channel = MethodChannel(
+      'purchases_flutter',
+      const StandardMethodCodec(),
+      registrar,
+    );
+
+    final instance = PurchasesFlutterPlugin();
+    channel.setMethodCallHandler(instance.handleMethodCall);
+
+    void injectScript() {
+      final script = html.ScriptElement()
+        ..src =
+            'https://unpkg.com/@revenuecat/purchases-js-hybrid-mappings@0.0.7-alpha.1/dist/index.umd.js'
```

Comment:
This is the new library we are using that contains the mappings. This allowed to remove a lot of logic from here.